layouts/shortcodes/api-docs-alert.html

```diff
@@ -0,0 +1,21 @@
+{{ $providerName := .Get 0 }}
+{{ $packagePath := printf "/docs/reference/pkg/%s" $providerName }}
+
+{{- /* Not all providers have resource docs, so we check to make sure they do before trying to link to them. */ -}}
+{{ if fileExists (printf "%s/_index.md" $packagePath) }}
+    <div class="note note-info">
+        <p>
+            <strong class="mr-1">
+                <i class="fas fa-info-circle mr-1"></i>
+                Building something with {{ .Get 0 }}?
+            </strong>
+            You probably want the <a href="{{ relref . $packagePath }}">resource documentation</a>.
+        </p>
+        <p>
+            This page documents the {{ .Get 0 }} API, but it doesn't go into much detail about how to use
```

Comment:
This feels a bit strong (in that it basically says, "this is not the droid you are looking for," and doesn't feel like it explains why you'd use this reference over the resource docs (or vice versa).

My suggestion would be to remove the "Building something with .... You probably want" and change this entire block to read more like:

"Note: The [resource documentation] describes the way to interact with [resource], including its inputs, outputs, and examples. This page documents the API specification for {{ .Get 0 }}, which provides additional details at the language-level."